envs/example/defaults.yml

```diff
@@ -219,7 +219,7 @@ glance:
   api_workers: 1
   registry_workers: 1
   logging:
-    debug: True
+    debug: False
```

Comment:
Instead of changing this default (which has no impact on our production deployments), the default in the role should be changed instead.